Makefile

```diff
@@ -118,13 +119,33 @@ $(BUILD_DIR)/$(PROJECT)-%: $(STATIK_FILES) $(GO_FILES) $(BUILD_DIR) deploy/cross
 	(cd `dirname $@`; shasum -a 256 `basename $@`) | tee $@.sha256
 	file $@ || true
 
+.PHONY: kokoro-cross
+kokoro-cross: $(foreach platform, $(SUPPORTED_PLATFORMS), $(KOKORO_BUILD_DIR)/$(PROJECT)-$(platform))
+
+$(KOKORO_BUILD_DIR)/$(PROJECT)-%: $(STATIK_FILES) $(GO_FILES) $(KOKORO_BUILD_DIR)
```

Comment:
You'll need to revise the values on lines 68-73

Reply:
ah yeah, I think I'll just remove
```
GO_VERSION_darwin_arm64 = 1.16beta1
CGO_ENABLED_darwin_arm64 = 0
```
since we should use a consistent version now and always enable CGO